userland/libtock/console.c

```diff
@@ -26,47 +26,69 @@ static void putstr_cb(
   if (putstr_head == NULL) {
     putstr_tail = NULL;
   } else {
-    putnstr_async(putstr_head->buf, putstr_head->len, putstr_cb, NULL);
+    int ret;
+    ret = putnstr_async(putstr_head->buf, putstr_head->len, putstr_cb, NULL);
+    if (ret < 0) {
+      // XXX There's no path to report errors currently, so just drop it
+      putstr_cb(0, 0, 0, NULL);
```

Comment:
That felt like a more substantial change and I didn't want to commit that to the putnstr_async interface - an issue for another day